force-app/main/default/classes/IBMWatsonDynamicModel.cls

```diff
@@ -60,4 +56,8 @@ public virtual class IBMWatsonDynamicModel {
     return ret;
   }
 
+  public Map<String, Object> getDynamicProperties() {
```

Comment:
is there any way we can prevent changing the following JSON:
```
{
  "id": "my_id",
  "foo": "bar"
}
```

into this:

```
{
  "id": "my_id",
  "additional_properties": {
    "foo": "bar"
  }
}
```

its a completely artificial nesting of JSON added for any fields not called out in the model that extends the dynamic model. I would prefer all properties to be dynamic rather than selecting a few to be dynamic then wrapping the rest in `additional_properties`.

Reply:
We already spoke on Slack a bit about this but I'll repeat here for visibility:

The problem right now is that to be able to access an `@AuraEnabled` property from Javascript, it needs to have a specific getter with the annotation. We can't write those in our models because we don't know ahead of time what the name or type of the dynamic properties will be.

Going through an `additional_properties` object isn't ideal, but it lets us get around that issue and access internal properties in a familiar way. I'll look into it further because I may be missing something about how to implement this, but for now I think this is at least a serviceable solution. If you have ideas though, feel free to reach out or submit a PR, because accessing dynamic properties top-level is ideal.

On the bright side, the newest commit at least gives the string representation we're looking for.